class.wp-cldr.php

```diff
@@ -309,7 +315,7 @@ public function flush_all_wp_caches() {
 	 * @return array An associative array where keys are WordPress locales and values are CLDR data items
 	 */
 	private function get_locale_bucket( $locale, $bucket ) {
-		if ( null === $locale ) {
+		if ( '' === $locale ) {
```

Comment:
we used to pass `null` for an empty locale, but i got tripped up on a type juggling issue so decided to go with an empty string instead.  yes should be `empty()`. will change.